agent/04_agent_configure.sh

```diff
@@ -290,9 +374,13 @@ if [ ! -z "${MIRROR_IMAGES}" ]; then
 
 fi
 
-if [[ "${NUM_MASTERS}" > "1" ]]; then
-  set_api_and_ingress_vip
-fi
+ if [[ "${NUM_MASTERS}" > "1" ]]; then
+    set_api_and_ingress_vip
+  fi
 
-generate_cluster_manifests
-generate_extra_cluster_manifests
+if [[ $NETWORKING_MODE == "DHCP" ]]; then
+  generate_install_config_agent_config
+else
+  generate_cluster_manifests
+  generate_extra_cluster_manifests
```

Comment:
The extra manifests function should be applied in any case, as it exercises the functionality of deploying the custom manifests